common/src/main/python/dlpx/virtualization/common/exceptions.py

```diff
@@ -77,7 +77,10 @@ def _remove_angle_brackets(type_string):
             return type_string.replace('<', '').replace('>', '')
 
         def _get_type_name(type_object):
-            if type_object.__module__ != '__builtin__':
+            # In py3 the builtins module will be named 'builtins', in py2 it will
```

Comment:
nit: multi line comments mostly follow this in sdk code -  one empty comment line followed by the actual comment line and then closed with another empty comment line. e.g on line 138.